### PR TITLE
Vertex alpha

### PR DESCRIPTION
**Pull request by dmitrii-z**
### PURPOSE
Resolves https://amdrender.atlassian.net/browse/RPRHOUD-43
Added support for displayOpacity in meshes, and lookup node to control color/opacity primvars

### NOTES FOR REVIEWERS
to use color/alpha vertex data, rpr_primvar_lookup node is implemented.

### CODE REVIEW

**Comment by ARozghon** [comment]
approved by QA

File: pxr/imaging/plugin/hdRpr/mesh.cpp [hunk 1]
```diff
@@ -614,11 +627,24 @@ void HdRprMesh::Sync(HdSceneDelegate* sceneDelegate,
                                 }
                             }
                         }
+
+                        if (!m_opacitySamples.empty()) {
+                            if (newPoint) {
+                                for (int sampleIndex = 0; sampleIndex < m_uvSamples.size(); ++sampleIndex) {
+                                    if (sampleIndex >= m_opacitySamples.size() || pointIndex >= m_opacitySamples[sampleIndex].size()) {
+                                        TF_WARN("Failed verification sampleIndex >= m_opacitySamples.size() || pointIndex >= m_opacitySamples[sampleIndex].size()");
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/mesh.cpp:635]: This is a very "developer-y" error message. Should it be a warning or debug?  Or better explained

**Review by bsavery — COMMENTED** [review]
Question about the error message which is code being a "warning"

File: pxr/imaging/plugin/hdRpr/mesh.cpp (hunk 1 above)
Comment (dmitrii-z) [reply pxr/imaging/plugin/hdRpr/mesh.cpp]: Actually, I just copy-pasted the code for displayColor... It's for debugging. I changed the message for a better explanation.

**Review by bsavery — APPROVED** [review]

**Pull request merged**